php-frontend/src/main/java/org/sonar/php/ini/PhpIniParser.java

```diff
@@ -22,28 +22,21 @@
 import com.google.common.collect.ArrayListMultimap;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ListMultimap;
-import com.google.common.io.Files;
 import com.sonar.sslr.api.RecognitionException;
-import java.io.File;
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
+
 import java.util.ArrayList;
 import java.util.List;
+
 import org.sonar.php.ini.tree.Directive;
 import org.sonar.php.ini.tree.PhpIniFile;
 import org.sonar.php.tree.impl.lexical.InternalSyntaxToken;
 import org.sonar.plugins.php.api.tree.lexical.SyntaxToken;
+import org.sonar.plugins.php.api.visitors.PhpFile;
 
 public class PhpIniParser {
 
-  public PhpIniFile parse(File file) {
-    String content;
-    try {
-      content = Files.toString(file, StandardCharsets.UTF_8);
-    } catch (IOException e) {
-      throw new IllegalStateException("Could not parse " + file, e);
-    }
-    return parse(content);
+  public PhpIniFile parse(PhpFile file) {
+          return parse(file.contents());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)